docs-next/src/content/docs/explainers/assertion-libraries.mdx

```diff
@@ -0,0 +1,14 @@
+---
+description: Popular libraries that work well with Mocha
```

Comment:
I think this page should include everything mentioned on the wiki homepage and just be titled "Related tools"

We can keep everything as a set of lists, but be sure to use L2 headings instead of L3 headings (`## Assertion...` instead of `### Assertion...`

I'll review the remaining tools to see which we should keep and which to remove now...